src/index.html

```diff
@@ -8,7 +8,9 @@
   <title>Colored blocks</title>
   <link rel="stylesheet" href="style.css">
 </head>
-<body>
-  <h1>Colored blocks</h1>
+<body style="margin: 0; padding: 0;">
+  <div style="background-color: red; height:100px;"></div>
+  <div style="background-color: green; height:100px;"></div>
+  <div style="background-color: blue; height:100px;"></div>
```

Comment:
why did you decide to use inline styles?

use a separate file with styles

Reply:
because common class will got only height:100px, and i should set background color for each.